components/cam/src/physics/cam/physics_types.F90

```diff
@@ -417,22 +392,19 @@ subroutine physics_update_main(state, ptend, dt, tend)
               ixo, ixo2, ixh, pcnst, state%lchnk, ncol)
     endif
    
-    if ( waccmx_is('ionosphere') .or. waccmx_is('neutral') ) then 
-      zvirv(:,:) = shr_const_rwv / rairv_loc(:,:,state%lchnk) - 1._r8
-    else
-      zvirv(:,:) = zvir    
-    endif
+    zvirv(:,:) = zvir    
```

Comment:
Are we getting rid of WACCM related codes? I am not familiar with WACCM, so I am not sue how WACCM simulations will be impacted by this change.